check_csr.rb

```diff
@@ -25,8 +25,7 @@ def sign_psk(csr, certname, value)
 
 def sign_csr(csr, certname)
   if ! csr.attributes.select { |a| a.oid == "ExtReq" }.nil?
-    # https://tickets.puppetlabs.com/browse/SERVER-1005
-    `puppet cert --allow-dns-alt-names --ssldir /etc/puppetlabs/puppet/ssl sign #{certname}`
+    # Everything is fine, tell puppetca to sign the CSR
```

Comment:
hmm, as said yesterday, it seems this part was actually working (when the tests were corrected), even if the puppetca fails right after the check_csr.

So this commit has not its place in here for now.